docs/director/03-tenancy.md

```diff
@@ -0,0 +1,37 @@
+#Tenancy
```

Comment:
This paragraph doesn't render well. Also, I miss some information about tenancy in general.